spec/mailers/previews/school_certifying_officials_mailer_preview.rb

```diff
@@ -13,6 +13,8 @@ def build
   private
 
   def recipients(facility_code)
+    return [] if facility_code.blank?
```

Comment:
What's going on here?

Side note, what happens when you send and email to an empty array of recipients?

Reply:
Via https://guides.rubyonrails.org/action_mailer_basics.html#previewing-emails we can generate previews of emails for content testing locally by going to http://localhost:3000/rails/mailers/

Reply:
This change makes it so that http://localhost:3000/rails/mailers/school_certifying_officials_mailer/build?facility_code=11000139 works without the `facility_code` param in the url

Reply:
In the [sidekiq job](https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/workers/education_form/send_sco_email.rb#L94) it checks if there are recipients before attempting to actually send an email

Reply:
Having 0 recipients for an email seems like a noteworthy thing. Maybe incrementing a StatsD metric or raising and exception.... Granted it's completely unrelated to this PR, and I'm not an expert on that is and isn't exceptional on this sco feature.

Reply:
Added StatsD metric to the sidekiq job